apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import ProjectCard from './ProjectCard';
+
+const styles = {
+  grid: {
+    padding: 10,
+    width: 1000
+  },
+  card: {
+    display: "inline-block",
+    padding: 10
+  },
+  labHeading: {
+    textAlign: "left",
+    fontSize: 24,
+    color: "#5b6770",
+    marginLeft: 10,
+    marginRight: 65,
+    borderBottom: "1px solid #bbbbbb",
+    paddingBottom: 10,
+    paddingTop: 40
+  }
+};
+
+const ProjectCardGrid = React.createClass({
+  propTypes: {
+    projects: React.PropTypes.array,
+    galleryType: React.PropTypes.string
+  },
+
+// Most recently edited projects should display 1st. This might not be needed dependent on whether the projects returned from the query are already sorted by recency or not. ****ASK DAVE****
+  sortByDate(projects) {
+
+    let sortedProjects = projects.sort((project1, project2) => new Date(project2.projectData.updatedAt) - new Date(project1.projectData.updatedAt));
```

Comment:
Nit: We try to keep lines in our js <= 80 characters as much as possible. We're not super strict about it, but really long lines like this should be broken up as possible.